RELEASE_NOTES

```diff
@@ -1,6 +1,40 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.7.0 (13 March 2025)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Bug fixes:
+* bot only reports moving to trash_bin when relevant (#292)
+
+Improvements:
+* add support for specifying that build job script is located in another
+  repository (#283)
```

Comment:
Hmm, why wrap this across a 2nd line, just make it a one-liner?

Reply:
80 character limit 😛